tests/eslint-rule-tester.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @fileoverview Helpers for tests.
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+"use strict"
+const eslintVersion = require("eslint/package.json").version
+const RuleTester = require("eslint").RuleTester
+const unofficialApis = require("eslint/use-at-your-own-risk")
+const globals = require("globals")
+
+// greater than or equal to ESLint v9
+exports.gteEslintV9 = +eslintVersion.split(".")[0] >= 9
```

Comment:
**suggestion:** We already have `semver` as a dependency, we could use it here 🤔

Reply:
I feel like the way it's written now is pretty simple too, but of course I can change it to that if you feel the need. :)

Reply:
ehehe, this is my simple mind preferering to read the following
```js
exports.gteEslintV9 = semver.satisfies(eslintVersion, '>=9')
```

over
```js
exports.gteEslintV9 = +eslintVersion.split(".")[0] >= 9
```

Reply:
> semver.satisfies('9.0.0-alpha.0', '>=9')
false

seems it does not allow prereleases?